src/types.ts

```diff
@@ -34,6 +34,7 @@ export type OptionalQueryClientConfig = {
   WS_HOST?: string;
   enableWebsocket: boolean;
   isStandalone: boolean;
+  debounceTimeAutoResize: number;
```

Comment:
Would you like to add a bit opf JSDoc here to explain the purpose of this key ? This could be a good place to specify the units (miliseconds) it uses.

Reply:
Done.